CHANGELOG.md

```diff
@@ -37,6 +37,8 @@ All notable changes for each version of this project will be documented in this
                 [summaryFormatter]="dateSummaryFormat">
             </igx-column>
         ```
+    - **Behavioral Change** - `Column Autosize` feature now does not handle templated headers where the first level children are sized based on parent like default `div` and etc. Autosizing for such headers will not result in change.
```

Comment:
Also maybe we should mention that now auto-sizing via the API also considers min/max widths as previously these constraints were applied only on double-click user interaction.